app/Models/Mship/Concerns/HasDiscordAccount.php

```diff
@@ -17,6 +17,10 @@ trait HasDiscordAccount
      */
     public function syncToDiscord()
     {
+        if (! config('services.discord.token')) {
+            return;
```

Comment:
Possibly worth logging something to Bugsnag or logging a critical error?

Reply:
Reality is this is only ever going to happen in testing. Its a cheap and lazy way of avoiding the actual code executing in the test suite rather than mocking it. FWIW, the same approach is done for CTS, Moodle and Helpdesk (only they check for the presence of a matching oauth client).